app/models/user.rb

```diff
@@ -455,6 +456,17 @@ def user_active?
     self.state != "suspended" && self.state != "disabled"
   end
 
+  def self.verified_ITSI_user?(login)
+    user = User.find_by_login(login)
+    imported_user = user.imported_user if user
+    importing_portal = imported_user.importing_portal if imported_user
+    #assuming for now ITSI portal will be the only entry in importing portal
+    if importing_portal && importing_portal.portal_url == APP_CONFIG[:import_site_url]
```

Comment:
Do you think this check agains the 'import_site_url' is necessary?  
Can't we just assume that if there is an imported user, then we return imported_user.is_verified?
The only way to import users is as an admin.